src/index.html

```diff
@@ -1,15 +1,196 @@
+            id="age"
+            name="age"
+            value="12">
+          <br>
+        </div>
+        <!-- date of birth -------------------------- -->
+        <div class="fieldset__item">
+          <label for="date">Full date of birth:</label>
+          <input
+            type="date"
+            id="date"
+            name="date">
+          <br>
+        </div>
+        <!-- accept checker -------------------------- -->
+        <div class="fieldset__item">
+          <label for="checkbox">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            id="checkbox"
+            name="checkbox"
+            required>
+        </div>
+      </fieldset>
+      <!-- PERSONAL INFORMATION FIELDSET ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <!-- email -------------------------- -->
+        <div class="fieldset__item">
+          <label for="email">E-mail:</label>
+          <input
+          type="email"
+          id="email"
+          name="email"
+          placeholder="email@example.com"
+          required>
+        </div>
+        <!-- password -------------------------- -->
+        <div class="fieldset__item">
+          <label for="password">Password:</label>
+          <input
+          type="password"
+          id="password"
+          name="password"
+          minlength="8"
+          required
+          autocomplete="new-password">
+        </div>
+      </fieldset>
+      <!-- INTERESTING FACTS FIELDSET ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <!-- love cats? -------------------------- -->
+        <div class="fieldset__item">
+          <label for="cats">Do you love cats?</label>
+          <input
+            type="radio"
+            id="cats"
+            name="cats">
+            Yes
+          <input
+            type="radio"
+            id="cats2"
+            name="cats">
+            No
+        </div>
+        <!-- favorite color? -------------------------- -->
+        <div class="fieldset__item">
+          <label for="color">What is your favorite color?</label>
+          <input
+            type="color"
+            id="color"
+            name="color"
+            value="#fff">
+        </div>
+        <!-- time to bad? -------------------------- -->
+        <div class="fieldset__item">
+          <label for="time">What time do you go to bad?</label>
+          <input
+            type="time"
+            id="time"
+            name="time">
+        </div>
+        <!-- brand of cars? -------------------------- -->
+        <div class="fieldset__item">
+          <label for="cars">What are you favorite brand of cars?</label>
+          <select
+            id="cars"
+            name="select"
+            size="4"
+            multiple>
+            <option value="BMW">BMW</option>
```

Comment:
переноси закрывающую скобку
```suggestion
          <select
            id="cars"
            name="select"
            size="4"
            multiple
          >
            <option value="BMW">BMW</option>
```